release/nightly_tests/dataset/data_ingest_benchmark.py

```diff
@@ -13,7 +13,7 @@
 GiB = 1024 * 1024 * 1024
 
 
-@ray.remote
+@ray.remote(num_cpus=0.5)
```

Comment:
Why 0.5? (Instead of 1, for example?)

Reply:
no particular reason, we just have to set it here. This actor is not computation intensive so make it a bit lower.